main.py

```diff
@@ -32,15 +31,19 @@ def cost(
     return 1 - circuit(params, rotations, masked_circuit)[0]
 
 
-def cost_iris(
+def cost_basis(
     circuit,
     params,
     data,
     target,
     rotations: List,
     masked_circuit: MaskedCircuit,
+    wires: int,
+    wires_to_measure: list[int],
```

Comment:
Afraid this only works since Python 3.9. 😞 
```suggestion
    wires_to_measure: List[int],
```